content/api/integrations/pagerduty.md

```diff
@@ -7,33 +7,34 @@ external_redirect: /api/#pagerduty
 
 ## PagerDuty
 
-Configure your Datadog-PagerDuty integration directly through Datadog API.  
+Configure your Datadog-PagerDuty integration directly through the Datadog API.
 [Read more about Datadog-PagerDuty integration][1]
 
-**Note**: 
+**Note**:
 
-* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization. 
+* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization.
 * Using the `PUT` method updates your integration configuration by **replacing** your current configuration with the new one sent to your Datadog organization.
+* All arguments are required in the payload **only** when creating (`PUT`) a new PagerDuty configuration.
```

Comment:
The wording "in the payload **only**" is confusing.  Specifically the words "payload" and "only". I'd just remove both, what do you think @cswatt ?

Reply:
👍